File: pulsar/src/videoDAO.py
class VideoDAO:
    conn = None

    def __init__(self, conn):
        self.conn = conn

    def update_status(self, slug, status):
        if self.conn is not None:
            cs = self.conn.cursor()
            cs.execute("UPDATE videos SET status = %s WHERE slug = %s;", (status, slug))

File: pulsar/pulsar/daos/VideoDao.cs
using System;
using pulsar.clients;

namespace pulsar.daos
{
    public class VideoDao
    {
        private ISqlClient _sqlClient;

        public VideoDao(ISqlClient sqlClient)
        {
            this._sqlClient = sqlClient;
        }

        public void UpdateStatus(string slug, string status)
        {
            this._sqlClient.Execute($"UPDATE videos SET status='{status}' WHERE slug='{slug}';");
        }
    }
}